packages/frontend/src/components/NavBar/ProjectSwitcher.tsx

```diff
@@ -168,14 +174,14 @@ const ProjectSwitcher = () => {
                         key={item.projectUuid}
                         onClick={() => handleProjectChange(item.projectUuid)}
                     >
-                        <Group>
-                            <Text>{item.name}</Text>
-
+                        <Group spacing="sm">
                             {item.type === ProjectType.PREVIEW && (
                                 <Badge color="blue" variant="filled" size="sm">
```

Comment:
not related to this bug but the badge could be way smaller 🤔

Reply:
updated!